core/app/run.go

```diff
@@ -97,6 +97,11 @@ func (v VersionSpec) GreaterThan(o VersionSpec) bool {
 	}
 }
 
+// Equal returns true if v is equal to o.
```

Comment:
Since this ignores the Build field, the comment should be updated to point that out.

Reply:
done